lab-jr/lib/greet.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function(name){
+  return 'Hello ' + name;
+};
```

Comment:
You have an `index.js` file, but it's not requiring in your module, and running the code. The correct way to run that would be `node index.js`, which should execute the code in your module.